src/WrappedMToken.sol

```diff
@@ -466,6 +504,8 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
 
         emit Transfer(sender_, recipient_, amount_);
 
+        if (amount_ == 0) return;
```

Comment:
won't happen if yield is checked instead of fee 🤔

Reply:
I added this for another reason as well. ERC20 requires explicit transfers of 0 token to still emit an event, so this allows that to happen _without_ wasting gas on all the code below it.